adding 9 icons

sebaseg-dev opened the pull request:
Adding 7 icons:
- Audiomack - drawn according to [official style guide](https://styleguide.audiomack.com/#!introduction)
- Bandsintown - [official brand assets](https://www.company.bandsintown.com/brand-assets)
- Bonfire - adapted from the [official brand assets](https://blog.bonfire.com/brand-assets-and-press-kit/) with their own recommendations and according to their favicon 
- Cameo - [official brand assets](https://brand.cameo.com/logos)
- Layers.to - [front page logo](https://layers.to/explore/trending)
- OpenSea - [official brand assets](https://docs.opensea.io/docs/logos)
- Ravelry - [official brand assets](https://www.ravelry.com/help/logos)

Other: - Fixing the AniList button

sebaseg-dev committed:
adding 9 buttons

sebaseg-dev committed:
relecture (fixing alt texts)

sethcottle commented:
Hey @sedomu, thanks for contributing this! Few comments we can chat through, would love your opinions!

---
### Audiomack

🤔 **Audiomack** mostly looks good to me. Might be worth adding the gradient to the logo, I noticed every orange example in the guidelines uses the gradient (but the official, .eps doesn't seem to support it when I download it). I also noticed some vector path clipping around the logo you're using, I downloaded their .eps from the guidelines you linked above and exported two new variations for you, one with just the orange, and one with a gradient if you'd like to use that.

![CleanShot 2024-04-02 at 19 46 36@2x](https://github.com/sethcottle/littlelink-extended/assets/1301949/4bb23a85-12c8-4060-8f73-8fc2c6e50601)

Orange: ![audiomack](https://github.com/sethcottle/littlelink-extended/assets/1301949/7d3d6d42-e89b-4421-895d-b056e6c972a1)
Gradient: ![audiomack](https://github.com/sethcottle/littlelink-extended/assets/1301949/f1c6252f-7cc0-4dd4-90bd-052966db3849)

![audimackgroup](https://github.com/sethcottle/littlelink-extended/assets/1301949/d583fb06-9373-4093-bf09-d05289fd477b)

See these with common vision impairment simulations:
![audiomack-visionsimulator](https://github.com/sethcottle/littlelink-extended/assets/1301949/1e36efa0-5b7e-4252-8035-ba09ff01e7a7)

---
### Bandsintown

✅ **Bandsintown** looks good to me.

---
### Bonfire

🤔 For **Bonfire**, there could be some more appealing color combinations that are displayed on in the official brand guidelines you've linked. Here's an example. These would allow for a larger 24x24 brand icon, instead of constraining it to a circle and still use the prominent brand colors. These are leveraging their Orange and Gray. I personally prefer gray background with orange logo option, because it is higher contrast. Both would need a black and white stroke for additional background contrast.

![bonfire](https://github.com/sethcottle/littlelink-extended/assets/1301949/5f19482b-86cc-4c43-8d64-3803f5e69e76)

White: ![bonfire](https://github.com/sethcottle/littlelink-extended/assets/1301949/324a1606-3fca-4e77-b165-b4d57191d8b6)
Orange: ![bonfire](https://github.com/sethcottle/littlelink-extended/assets/1301949/2bf7bb6b-337a-4270-92a7-ad8615c55ff6)

See these with common vision impairment simulations:
![visionsimulator](https://github.com/sethcottle/littlelink-extended/assets/1301949/af9d1d2a-ce68-408b-a747-d83d09dcfeea)

---
### Cameo

✅ **Cameo** looks good to me.

---
### Layers

✅ **Layers.to** looks good to me.

---
### OpenSea

🤔 For **OpenSea**, a lot of their documentation site seems to leverage the Sea Blue as the primary button color. You could use the transparent logo with that color combination? It may be a good option to help brighten up the button availability. Though here are a few others based on their [badge designs](https://docs.opensea.io/docs/badges) (including the default text they use)—subjective, but if I was just using black or white, I would use the white. Seems to pair a bit better with their Sea Blue.

![opensea](https://github.com/sethcottle/littlelink-extended/assets/1301949/dd01f9ef-c597-4e6e-b3c1-d6e4e0430f88)

See these with common vision impairment simulations:
![opensea-visionsimulator](https://github.com/sethcottle/littlelink-extended/assets/1301949/35ac4793-c35f-4bb5-b6da-3c39e22d5b71)

---
### Ravelry

🤔 For **Ravelry**, the stroke will need adjusted around the current implementation to make sure it's accessible on light theme (it's currently `#F3F4F0`, the same as the background color). I would maybe consider using black text, as their red is not very accessible on the "green" background for those with visual imparements.

![raverly](https://github.com/sethcottle/littlelink-extended/assets/1301949/608b9943-c914-4dcf-83d2-d1e75d9c3604)

See these with common vision impairment simulations:
![Vision Simulations](https://github.com/sethcottle/littlelink-extended/assets/1301949/0fa48808-0404-4063-b4d8-fdbe68c04525)

---

### AniList

🤔 For **AniList**, I think we have two possible options, instead of pure black. Their website navigation uses `#2B2D42` as the background color. Their footer uses a darker `#11161D` which looks to be what their site favicon is maybe using (so it could be the best option).

![anilist](https://github.com/sethcottle/littlelink-extended/assets/1301949/c5cd2887-6732-4efa-b19d-95e5e99b9218)

See these with common vision impairment simulations:
![anilist-visionsimulator](https://github.com/sethcottle/littlelink-extended/assets/1301949/5c22d9ef-eb66-4c25-8b76-359aeed5d76b)

sebaseg-dev committed:
adding Big Cartel and Gumroad buttons
adding gradient to Audiomack svg (from homepage logo)
redesigning Bonfire logo
changing OpenSea's colours
changing ravelry's colours
changing AniList's colours

sebaseg-dev commented:
Hey @sethcottle ,
I worked on the changes you proposed.
About accessibility, first I'm personnaly partially colour blind (I don't see a significant part of the blue spectrum), meaning I can't appreciate constrast the same way. To explain changes I opted for:
- Audiomack: I checked many times on their guidelines for this colour gradient and concluded it was just a visual effect due to the variable thickness of the line. Finally, I just took their logo on their frontpage to get the "true" gradient. Works fine. I didn't saw neither the holes on the previous proposition: nice job!
- Bonfire: I was focusing on the fact we don't use their font, so I resctricted myself to use their favicon, but you're right. For the colours, I chose the orange one: this way, both logotype and text are pure white, which is allowed by their guidelines, whereas using 2 different colours is explicitely not allowed.
- OpenSea: that's a great example for my colour blindness. I totally agree to use their colours. But to me, It looks really faded and more difficult to read. That's why I used the "default" colours combination. But I trust you, with my condition, I'm not the "main" usercase.
- Ravelry: I agree for black letters for another reason. If I'm not mistaken, every other piece of text on LittleLink is either pure black or pure black. So, there is no reason why ravelry would be unique. And it's not forbidden by their guidelines.
- AniList: I prefer the navigation colour by far.

Thanks for your feedback, it really enhances the whole!

Before all these design choices, I think it could be helpful to precise a graphical charter with some basic and measurable framework rules to define the style to use. Later I might think in more detail about that and may propose something, if you think that's relevant.

sethcottle commented:
Thanks again @sedomu! I'm going to get this merged. Thanks for also bringing in the Gumroad and Big Cartel commit into this one. As for the framework, I think that would be a great idea, happy to hear any proposals and I can also spend a little time working through that soon, I hope.
